src/components/SignInPage/Form.js

```diff
@@ -0,0 +1,12 @@
+import InputBox from '../InputBox';
+
+function Form(){
```

Comment:
This Form component is similar to the Sign Up page Form. Lets merge them into one component that takes in params for what input boxes are needed. Lets call it authentication form.

Reply:
I created the auth form component which takes an array of strings for the input box placeholder and another parameter that takes the message for the button.